aten/src/ATen/native/ReflectionPad.cpp

```diff
@@ -922,154 +372,35 @@ Tensor reflection_pad2d_backward_cpu(
     const Tensor& grad_output,
     const Tensor& input,
     IntArrayRef padding) {
-  auto grad_input = at::zeros_like(input, LEGACY_CONTIGUOUS_MEMORY_FORMAT);
+  auto grad_input = at::empty({0}, input.options());
```

Comment:
I see you're resizing the empty grad input. Maybe you mean to use empty_like?

Again I'm not sure whether this is a good change for autograd. cc @albanD

Reply:
updated!